### ISSUE TITLE
how to print coverage model

### ISSUE THREAD

**Issue by songzenghui** [fvutils/pyvsc#181]
I want to print my coverage model to confirm it met my exceptations. I use `get_coverage_report_model`, but the `CoverageReport` does not implement `__str__` or `__repr__`. 
Could you implement that to print the coverage model?

**Comment by mballance** [comment]
Hi @songzenghui,
Let me give that some thought. In general, it's a good idea to have Python objects be easily converted to a string representation.
Currently, you can use `get_coverage_report` to get a string representation of the coverage. You can also use report_coverage to write the report to a file. See the information here:
[Coverage Reports](https://pyvsc.readthedocs.io/en/latest/coverage.html#coverage-reports)

**Comment by songzenghui** [comment]
Yes, I use this api. But why it print two covergroup, actually i build one covergroup model.
![image](https://github.com/fvutils/pyvsc/assets/25813311/6293f7e8-eed9-4ecc-ab4c-04e1d4d826c4)
Look, I want to print one. why there is `TYPE` and `INST`

**Comment by mballance** [comment]
Hi @songzenghui,
Okay, so it sounds like you'd like to be able to selectively display portions of the coverage model, and having __str__/__repr__ on the coverage model will help with that.
To your specific question about Type vs Instance coverage, coverage tools generally maintain coverage information on a per-covergroup-type basis as well as a per-covergroup-instance basis. When you only have a single instance of a covergroup, the Type and Inst information is the same. I decided to keep the format of the coverage report the same, instead of changing the format depending on whether a single or multiple instances of a covergroup had been created.
